src/oemof/solph/components/_offset_converter.py

```diff
@@ -365,16 +366,14 @@ def __init__(
         inputs,
         outputs,
         label=None,
-        conversion_factors=None,
-        normed_offsets=None,
+        coefficients=None,
```

Comment:
Is the deletion and addition here intended? Or some kind of relict from a merge?

Reply:
Eventually, I made a mistake when merging. That would mean that i also made another one when reviewing #1068. (The `OffsetTransformer` as a transitional wrapper should offer the old API, shouldn't it?)

Reply:
It does: https://github.com/oemof/oemof-solph/blob/80298509a7e8e485cd2b2c35cf2fcfa0fcfbf399/src/oemof/solph/components/_offset_converter.py#L117-L119, there also is a test: https://github.com/oemof/oemof-solph/blob/80298509a7e8e485cd2b2c35cf2fcfa0fcfbf399/tests/test_components/test_offset_converter.py#L420

Reply:
And, in this branch it is also available: https://github.com/oemof/oemof-solph/blob/revision/refine_timeindex/tests/test_components/test_offset_converter.py

Therefore, I guess that those keywords do not have any effect (!?)...

Reply:
As the keywords `conversion_factors` and `normed_offsets` are not part of the old API, I thought they were in the merge by mistake (or because I changed something). In my opinion, it does not make sense to have an `OffsetTransformer` with these keywords.

Reply:
Ah that is on the transformer... I overlooked that, then all is fine! :)